app_dart/lib/src/model/appengine/task.dart

```diff
@@ -262,6 +265,11 @@ class Task extends Model<int> {
     _status = value;
   }
 
+  /// Whether the task is a benchmark test
+  @BoolProperty(propertyName: 'Benchmark')
+  @JsonKey(name: 'Benchmark')
+  bool? isBenchmark;
```

Comment:
We don't want to add this here. Instead we should be adding a Map<String, dynamic> properties (the equivalent of what's in models/ci_yaml/target.dart -> getProperties())

Reply:
This is to save the benchmark flag to datastore. Do you want to save the entire properties? which I don't think is needed and will make the query complicated. Or do i miss anything?

Reply:
Yes, let's save the entire properties. This should be straight forward by passing it from Target to Task.

Part of the migration to microservices is to simplify what data we store in entities. Adding columns to the datastore isn't sustainable when only a few tests would benefit from it. \cc @yusuf-goog is likely to remove a lot of these entries in the results processing refactor

Reply:
Thanks for the explanation. That makes sense to me. 
@KristinBi since datastore doesn't support map (https://cloud.google.com/datastore/docs/concepts/entities#properties_and_value_types), you may need to save the properties as string via jsonEncode, and use jsonDecode when retrieving the values.